### ISSUE TITLE
"Capitalization doesn't help very much" can be confusing

### ISSUE THREAD

**Issue by vita10gy** [dropbox/zxcvbn#116]
"Capitalization doesn't help very much" can be confusing in situations where we're also asking users to use a capital letter.

Would it be possible to use something more along the lines of "Capitalizing the first letter in your password doesn't help very much"?

**Comment by PythonNut** [comment]
Perhaps "Capitalizing words doesn't help very much."

**Comment by ZandercraftGames** [comment]
Sorry to jump in here literally 10 years later (although I'm glad someone took the same issue with the wording as I did), but another thing to consider is that other than being a little obtuse and hard for users to understand, "Capitalization doesn't help very much" doesn't sound very professional.

Something like "Capitalizing words does not greatly impact the password's security". I know it's a little wordy, but I'm sure there's a better way to get that message across.

**Comment by vita10gy** [comment]
> Sorry to jump in here literally 10 years later (although I'm glad someone took the same issue with the wording as I did), but another thing to consider is that other than being a little obtuse and hard for users to understand, "Capitalization doesn't help very much" doesn't sound very professional.
> 
> Something like "Capitalizing words does not greatly impact the password's security". I know it's a little wordy, but I'm sure there's a better way to get that message across.

I can't believe that was a decade ago.

**Comment by ZandercraftGames** [comment]
> I can't believe that was a decade ago.

Haha time flies! :)

**Comment by ZandercraftGames** [comment]
Not sure if you guys have seen, but someone made a better maintained TS fork at https://github.com/zxcvbn-ts/zxcvbn

**Comment by KunalTanwar** [comment]
Agreed — fixed in [zxcvbn-ts](https://github.com/KunalTanwar/zxcvbn-ts)! The suggestion has been reworded to "Capitalizing the first letter is a common pattern and doesn't add much security", which is more specific and less likely to conflict with password policy requirements asking for a capital letter.
